testdata/asm/gf251_add.zkasm

```diff
@@ -0,0 +1,22 @@
+;; GF251 curve
+;; Add on u256
+;; Mul on u8, to be able to test all cases
+
+fn add(x u256, y u256) -> (RESULT u256) {
```

Comment:
I don't understand why you have `add(u256,u256) -> u256`.  Should be `add(u8,u8)->u8`.